frontend/src/app/components/ExperimentRowCard.js

```diff
@@ -79,6 +86,32 @@ export default class ExperimentRowCard extends React.Component {
     );
   }
 
+  renderFeedbackButton() {
+    if (!this.props.enabled) { return ''; }
```

Comment:
Hmm, I thought I saw an error from React when I used null, but that might have been something else